src/masternodes/icxorder.cpp

```diff
@@ -0,0 +1,248 @@
+#include <masternodes/icxorder.h>
+
+/// @attention make sure that it does not overlap with other views !!!
+const unsigned char CICXOrderView::ICXOrderCreationTx           ::prefix = '1';
+const unsigned char CICXOrderView::ICXOrderKey                  ::prefix = '2';
+const unsigned char CICXOrderView::ICXOrderStatus               ::prefix = 'B';
```

Comment:
Duplicate, you should recheck with existing prefixes.

Reply:
Didn't check overlap after rebase with 1.7.x. I will change them.